src/output.cpp

```diff
@@ -174,7 +174,7 @@ void trim_and_print(WINDOW *w, int begin_y, int begin_x, int width, nc_color bas
             iLength += iTempLen;
 
             if ( iLength > width ) {
-                sTempText = sTempText.substr(0, cursorx_to_position(sTempText.c_str(), iTempLen - (iLength - width) - 1, NULL, -1)) + "…";
+                sTempText = sTempText.substr(0, cursorx_to_position(sTempText.c_str(), iTempLen - (iLength - width) - 1, NULL, -1)) + "...";
```

Comment:
@illi-kun Thank you reading my changes. When using the single length character "…", Visual Studio gives a compile error.